app/controllers/users_controller.rb

```diff
@@ -26,4 +26,16 @@ def show
         @resume_url = @user.resume.url
     end
   end
+
+  def edit
+    @user = User.find(params[:id])
+    @profile_image_url = nil
```

Comment:
You should have something here like `@skills = Skill.all`, so you can pass all the skills in as a component. Otherwise, you can directly do it in the view by passing in `skills: Skill.all`